operations/availabilityblocks.md

```diff
@@ -196,7 +197,8 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `UpdatedUtc` | string | required | Last update date and time of the block in UTC timezone in ISO 8601 format. |
 | `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
 | `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `ReleasedUtc` | string | required | The moment when the block and its availability is released in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | optional | The moment when the block and its availability is released in UTC timezone in ISO 8601 format. Mutually exclusive with `RollingReleaseOffset`; the block will not be automaticaly released if neither `ReleasedUtc` nor `RollingReleaseOffsetUtc` is specified. |
```

Comment:
I think the change is perfectly well described here. However, I am concerned that changing `ReleasedUtc` from `required` to `optional` is a potentially breaking change. Has this been agreed with the API Team?

Reply:
Answered here: https://github.com/MewsSystems/gitbook-connector-api/pull/641/files/5cc5cd3ac52eb8b54ae60bb82ff7d9f06e970caf#r1745350934